tdp_lib_dag/firewall.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: firewall_generic_config
+  depends_on: []
```

Comment:
If you do not depends on `tdp-cluster_config` the firewall will not be restarted when port are changed in `tdp-cluster` `tdp_vars`.

Reply:
@mralbertk can you handle this?

Reply:
Done in https://github.com/TOSIT-IO/tdp-collection-extras/pull/103/commits/08f7f15112185827c8779a3dfa93783357ec4058